backend/infrahub/webhook/models.py

```diff
@@ -81,12 +81,16 @@ class TransformWebhook(Webhook):
     transform_class: str = Field(...)
     transform_file: str = Field(...)
 
-    async def _prepare_payload(self) -> None:
+    async def _prepare_payload(self, service: InfrahubServices) -> None:
```

Comment:
Could we use self.service here instead of extra parameter? It would also avoid above `# pylint: disable=unused-argument`

Reply:
Good catch, I missed that it was already present on the base class